jobs/build/ocp/Jenkinsfile

```diff
@@ -104,13 +107,16 @@ RPMs:
     Exernal Mirror: ${mirrorURL}/${OCP_PUDDLE}
 ${image_details}
 
+${timing_report}
```

Comment:
I think these should be built into image_details so we don't proliferate whitespace. If image_details is empty, these will be too (6 blank lines between `External Mirror:` and `Brew:`).